app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -19,7 +19,7 @@ def send_rrd_alert_email(subject, message)
   end
 
   # @param metadata_hash [Hash] to be merged into form_json['rrd_metadata']
-  def add_metadata(metadata_hash)
+  def save_metadata(metadata_hash)
```

Comment:
Rename for accuracy (since it's saving to the DB) and to distinguish from `ClaimContext.add_metadata`